src/entities/platforms.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+from entities.class_utils import AbstractDataClass
+from entities.interfaces import IScorable, ISerializable
+from enum import Enum
+from dataclasses import dataclass, field
+from entities.enumerations import Color
+from entities.scoring_elements import Ring, Goal
+from entities.robots import Robot
+
+
+class PlatformState(int, Enum):
+    LEFT = 0
+    RIGHT = 1
+    LEVEL = 2
+
+
```

Comment:
I enjoyed reading about [data classes](https://towardsdatascience.com/9-reasons-why-you-should-start-using-python-dataclasses-98271adadc66) and think this is a good job implementing. 

I'm going to posit we have the accessors we need and we'll just add others if it gets too annoying 🙌

Reply:
Definitely. We can override the provided accessors if needed, as well as add other methods as needed. Inheritance gets somewhat messy with dataclasses (as it likely would with normal classes) which is the driving reason behind passing around kwargs (since elements can be instantiated with some default information if they're "new", but adding them in a random game state may make it nice to do something such as add rings to a high branch on a goal)